src/giterm/giterm.py

```diff
@@ -124,12 +127,14 @@ def _main():
         description='''A terminal-based GUI client for Git.''')
     parser.add_argument(
         '-v', '--version', action='version', version=__version_text__)
-    parser.parse_args()
+    parser.add_argument('repo',
+        nargs='?', help='(Optional) Path to git repository; will default to cwd if not given.')
```

Comment:
Great idea, though I liked the ability to `giterm` from anywhere in the working copy. The new behavior implies being at the top-level directory of the working copy or it won't work at all. Would you please make both behavior compatible?